include/extractor/guidance/validation_handler.hpp

```diff
@@ -46,7 +46,8 @@ class ValidationHandler final : public IntersectionHandler
                             const EdgeID via_eid,
                             Intersection intersection) const override final
     {
-        checkForSharpTurnsOntoRamps(nid, via_eid, intersection);
+        // checkForSharpTurnsOntoRamps(nid, via_eid, intersection);
```

Comment:
eventually the validation handler should store deduplicated

    (intersection, category)

tuples and on destruction report them as geojson with properties attached to the features.

For now I think this is fine.